### ISSUE TITLE
Support for widgets

### ISSUE THREAD

**Issue by mmontminy** [zenyagami/car-dashdroid#2]
Something I've always wondered about car launcher apps is why there's so much effort put into trying to get them to work well with various music apps, rather than adding support for placing an app's widget on the launcher screen.

Most music players, podcast players, etc, all provide a 4x2 widget, which seems like it would fit nicely in place of the current media controls.

Aesthetically, it may not fit in with the design well, but I'd prefer to have the enhanced functionality a widget offers.

Bonus points if support for "swiping" between different widgets were supported. I'm either listening to Google Play Music, or Pocket Casts, depending on mood. If I could replace the current media player with both widgets, and switch between them with a simple gesture, that would be ideal.

**Comment by zenyagami** [comment]
Yeah I will add gestures, and I can't place widgets due android limitations,I can't use widgets inside my app, that would be awesome but I can't android don't let me U.u

**Comment by mmontminy** [comment]
I'm no developer, so I'll take your word for it, but it seems to me car home apps are launchers similar to home screen launchers. I could swear I've seen apps that embed widgets before. I thought I recently heard of an app that displayed widgets in a floating pop-up window.

 I also found this http://www.xda-developers.com/learn-how-to-add-widget-support-in-your-app/

**Comment by zenyagami** [comment]
oh snap!!! thanks i will take a look that will solve a lot of issues :P
